qa/workunits/cephtool/test.sh

```diff
@@ -562,6 +562,7 @@ function test_auth()
   ceph auth caps client.xx osd "allow rw"
   expect_false sh <<< "ceph auth get client.xx | grep caps | grep mon"
   ceph auth get client.xx | grep osd | grep "allow rw"
+  ceph auth caps client.xx mon 'allow command "osd tree"'
```

Comment:
this test passes even without your change. could you add a test in `src/test/confutils.cc` ?